src/main/java/de/hpi/bpt/argos/common/ObservableImpl.java

```diff
@@ -0,0 +1,75 @@
+package de.hpi.bpt.argos.common;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * {@inheritDoc}
+ * This is an abstract base implementation.
+ */
+public class ObservableImpl<Observer> implements Observable<Observer> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00119)